examples/multimodal/utils.py

```diff
@@ -72,6 +72,13 @@ def add_common_args(parser):
         action='store_true',
         help='Enables chunked context (only available with cpp session).',
     )
+    parser.add_argument(
+        '--ptable_offloading',
```

Comment:
may add a section in multimodal/README.md to briefly introduce the idea -- you can reuse the PR description in the README